nav2_smac_planner/lattice_primitives/trajectory_generator.py

```diff
@@ -566,3 +569,39 @@ def generate_trajectory(
         trajectory_path = self._create_path(trajectory_params, primitive_resolution)
 
         return Trajectory(trajectory_path, trajectory_params)
+
+if __name__ == "__main__":
+    test = TrajectoryGenerator({"turning_radius":0.5})

```

Comment:
Should this be here?

Reply:
removed it in latest. Forgot to remove my testing code...